setup/install/providers/docker-registry.md

```diff
@@ -158,7 +158,7 @@ API](https://console.developers.google.com/apis/api/cloudresourcemanager.googlea
    ```bash
    hal config provider docker-registry account add my-docker-registry \
     --address $ADDRESS \
-    --username $USERNAME \
+    --username _json_key \
```

Comment:
Could you remove the assignment of `USERNAME=_json_key` on line [142 above as well](https://github.com/spinnaker/spinnaker.github.io/pull/950/files#diff-3b34abaaf580b7c9636d788f0756680fR142)?

Reply:
Done